docs/40_node-operation/10_getting-started/60_logging/10_native_logging.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Native Logging
+---
+
+The logging mechanism for `nodeos` is controlled through the `logging.json` file also known as logging configuration file. The logging configuration file can be used to define [appenders](#appenders) and to tie them to [loggers](#loggers) and [logging levels](#logging-levels).
+
+## Configure logging.json File
+
+The `logging.json` file is usually located in the specified `--config-dir`, the same directory as the `config.ini` file. This path can be explicitly defined using the `-l` or `--logconf` options when starting `nodeos`.
+
+```sh
+    nodeos --help
+```
+
+```console
+    ...
+    Application Command Line Options:
+    ...
+    --config-dir arg                      Directory containing configuration files such as config.ini
+    -l [ --logconf ] arg (=logging.json)  Logging configuration file name/path for library users
+```
+
+## Appenders
+
+The EOS logging library supports two `appender` types:
+
+- [GELF](#gelf) (Graylog Extended Log Format)
+- [Console](#console)
+
+### GELF
+
+This `appender` sends the log messages to `Graylog`. `Graylog` is a fully integrated platform for collecting, indexing, and analyzing log messages. The supported configuration options are:
```

Comment:
Might as well link externally to them: https://www.graylog.org/

Or maybe github is better? https://github.com/Graylog2/graylog2-server

Reply:
I like the second one